### PR TITLE
Lab 1

### PR DESCRIPTION
**Pull request by ChumakAn**

### CODE REVIEW

File: com/company/packetOfMilk.java [hunk 1]
```diff
@@ -0,0 +1,187 @@
+package com.company;
+
+class packetOfMilk {
```
Comment (MrSpodaryk) [inline com/company/packetOfMilk.java:3]: wrong code convention
why your class is not public?

File: com/company/packetOfMilk.java [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    private static int shelfLifeMonths = 0;
+    private String manufacturer = "none";
+    private int fatness = 0;
+    private int volumeInMll = 0;
+    private int caloriesPer100Grams = 0;
+    private String typeOfMilk = "default";
+
+    private int sweetnessInPercents = 0;
+    private String toppings = "default";
+    private float price = 0;
```
Comment (MrSpodaryk) [inline com/company/packetOfMilk.java:13]: better use double

File: com/company/packetOfMilk.java [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+    private String lactose = "default";
+    private String typeOfCarton = "default";
```
Comment (MrSpodaryk) [inline com/company/packetOfMilk.java:15]: set default values in constructor or initialization block

File: com/company/Main.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    public static void main (String []args)
+    {
+        packetOfMilk selyanochka =  new packetOfMilk();
+        System.out.println(selyanochka.toString());
+        packetOfMilk molokiya = new packetOfMilk("MOLOKIYA", 12 , 500, 376);
+        System.out.println(molokiya.toString());
+        packetOfMilk prostokvashyno = new packetOfMilk("Prostokvasha",15, 890 ,
+                499, "CowMilk", 0, "none",
+                28.75F, "none", "eco", 21, 4.2F);
+        System.out.println(prostokvashyno.toString());
```
Comment (MrSpodaryk) [inline com/company/Main.java:13]: try don't call toString and see what will happen

File: com/company/Main.java [hunk 5]
```diff
@@ -0,0 +1,17 @@
+package com.company;
+import  com.company.packetOfMilk;
+public class Main {
```
Comment (MrSpodaryk) [inline com/company/Main.java:3]: don't name class like Main

File: com/company/packetOfMilk.java (hunk 1 above)
Comment (ChumakAn) [reply com/company/packetOfMilk.java]: fixed

File: com/company/packetOfMilk.java (hunk 2 above)
Comment (ChumakAn) [reply com/company/packetOfMilk.java]: fixed

File: com/company/packetOfMilk.java (hunk 3 above)
Comment (ChumakAn) [reply com/company/packetOfMilk.java]: fixed

File: com/company/Main.java (hunk 4 above)
Comment (ChumakAn) [reply com/company/Main.java]: fixed

File: com/company/Main.java (hunk 5 above)
Comment (ChumakAn) [reply com/company/Main.java]: fixed

File: com/company/packetOfMilk.java (hunk 1 above)
Comment (MrSpodaryk) [reply com/company/packetOfMilk.java]: so why your class was not public?

**Pull request merged**